src/services/services.ts

```diff
@@ -3113,7 +3114,9 @@ namespace ts {
             }
             else if (isRightOfOpenTag) {
                 let tagSymbols = typeChecker.getJsxIntrinsicTagNames();
-                if (tryGetGlobalSymbols()) {
+                // If the currect cursor is inside JSX opening tag, the only meaningful completions are those of JSX.IntrinsicElements or users defined React.Component
+                // If the services can't find those symbols, then show nothing instead of including all the global symbols in the completion list.
+                if (tryGetGlobalSymbols(/*includeAllGlobalSymbols*/false)) {

```

Comment:
What does "all" mean? Something is getting filtered out and this parameter name doesn't convey it well